db/migrate/20170711102511_create_ticket_scannings.rb

```diff
@@ -0,0 +1,9 @@
+class CreateTicketScannings < ActiveRecord::Migration
+  def change
+    create_table :ticket_scannings do |t|
+      t.integer :physical_ticket_id, null: false
+
+      t.timestamps null: false
```

Comment:
@Ana06 You mean with "registration" the moment when a user's ticket for accessing to the conference is used, right? Then the answer will be YES